globalsat_sport.cc

```diff
@@ -0,0 +1,742 @@
+/*
+    Globalsat Sport serial protocol.
+
+    Copyright (C) 2012, Zingo Andersen
+
+    Dual Licese Public Domain and GPLv2 or later
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program; if not, write to the Free Software
+    Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111 USA
+
+ */
+
+/*
+ * This is the bridge between the GPSBable and globalsport device
+ * gh625XT
+ *
+ *
+ * ./gpsbabel -i globalsat -f /dev/ttyUSB0 -o gpx,garminextensions -F xxx.gpx
+ * ./gpsbabel -i globalsat -f /dev/ttyUSB0 -o gtrnctr,course=0,sport=Running -F xxx.fit
+ *
+ * ./gpsbabel-debug -i globalsat -f /dev/ttyUSB0 -o gpx,garminextensions -F xxx.gpx
+ */
+
+
+#include <ctype.h>
+#include <limits.h>
+#include <endian.h>
```

Comment:
<endian.h> isn't a standard header.  We have be_read_[float,double] and friends like be_write16() in defs.h